### PR TITLE
df : Add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/common/df.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- 사람이 읽을 수 있는 형태로 모든 파일시스템들과 그들의 디스크 사용량 출력:
+
+`df -h`
+
+- 주어진 파일이나 디렉토리를 포함하는 파일 시스템과 그것의 디스크 사용량 출력:
+
+`df {{경로/파일_혹은_디렉토리명}}`
+
+- 사용 가능한 inode들의 수에 대한 통계 출력:
```
Comment (einverne) [inline pages.ko/common/df.md]:
```suggestion
- 사용 가능한 inode 들의 수에 대한 통계 출력:
```
A space between `inode` and other words looks better. :)

Comment (Proscream) [reply pages.ko/common/df.md]: Umm... In the md file, inode is inodes, the plural noun.  Korean '들' means, plural noun like ~s(es). 
Maybe its better "inode들의" or "inodes 수에 대한 통계 출력".
what do you think?

Comment (zdroid) [reply pages.ko/common/df.md]: Is it normal in Korean not to put spaces after a word in similar cases?

Comment (Proscream) [reply pages.ko/common/df.md]: Yes. It is like writing `inode s` in English.

Comment (einverne) [reply pages.ko/common/df.md]: Sorry, I am not familiar with the Korean, I am not sure how to put the space between the English words and Korean words. So let other Korea native speak to review.

**Pull request merged**